Ideas-2019.md

```diff
@@ -442,7 +442,7 @@ As with other Sugarizer activities, the new Write activity should be written usi
 
 ------------
 
-## Port Scratch 3.0 to Sugar Desktop and Fix sugar-web
+## Port MB & Scratch 3.0 to Sugar Desktop and Fix sugar-web
```

Comment:
To me, the biggest issue is fixing Sugar Web and making Sugarizer activities once again compatible with Sugar.
MB and Scratch are secondary goals.
With any remaining time, all of the existing Sugar Web/Sugarizer apps could be reviewed and ensured to work.
Any remaining time after that could be added some new apps... I have lots of ideas for those.

Reply:
So, ## Fix Sugar Web, port Scratch 3.0 and Music Blocks, and ensure other web activities are working